webapp/src/css/badge.less

```diff
@@ -42,9 +42,22 @@
   top: -3px;
 }
 
-.green-dot {
-  color: @green;
+.mm-svg-ok {
+  svg {
+    width: 14px;
+    height: 14px;
+    path {
+      fill: @ok-color;
+    }
+  }
 }
-.red-dot {
-  color: @red;
+
+.mm-svg-error {
+  svg {
+    width: 14px;
+    height: 14px;
+    path {
+      fill: @error-color;
+    }
+  }
```

Comment:
We've moved away from prefixing everything with `mm` because it's too common. Also there's too much repetition here. How about...

```css
.verification-badge {
  svg {
    width: 14px;
    height: 14px;
  }
  &.verified svg path {
    fill: @ok-color;
  }
  &.error svg path {
    fill: @error-color;
  }
}
```

The span will then have two classes: "verification-badge" and either "verified" or "error".